lib/src/internal/exceptions/unknown_enum_value.dart

```diff
@@ -0,0 +1,13 @@
+/// Thrown when a parsing method of an enum failed.
+class UnknownEnumValue implements Error {
```

Comment:
Prefer extending `Error` over implementing it so that stack traces are accurate.